CampaignConfig/mdc2020_beam.cfg

```diff
@@ -308,7 +308,7 @@ executable_2.name = generate_fcl
 executable_2.arg_4 = --aux=1:physics.filters.beamResampler.fileNames:inputs.txt
 executable_2.arg_5 = --events-per-job=400000
 executable_2.arg_6 = --run-number=%(run_number)s
-executable_2.arg_7 = --njobs=%(njobs)s
```

Comment:
Can you do math in POMS?  Ie could you have a global scale variable and then have the specific number of jobs be a scaled version of that?  100 X N, where N is the percent of the production?

Reply:
Not really. An alternative is to declare a bash variable that is then used as argument. Something like this for each stage:
`job_setup.prescript = let this_njobs=%(njobs)s*%(multiplier)s`
and then
`executable_2.arg_7 = --njobs=${this_njobs}`
which I don't like too much...

Reply:
How do you plan to scale this up for 10%, 100% production?  Doesn't it require hand-editing the configs?  That will be error-prone.
We can let this in as a bug fix, but we need to be thinking about the next set of jobs too.

Reply:
I agree. My idea was to have a global variable `%(njobs)s` set to the number of POT jobs and then scale it as showed for the downstream stages.